Earthfile

```diff
@@ -348,6 +348,13 @@ base-image:
         COPY github.com/kairos-io/immucore:$IMMUCORE_DEV_BRANCH+dracut-artifacts/10-immucore.conf /etc/dracut.conf.d/10-immucore.conf
     END
     # END
+    IF [ "$FLAVOR" = "ubuntu-20-lts" ] || [ "$FLAVOR" = "ubuntu" ] || [ "$FLAVOR" = "ubuntu-22-lts" ]
```

Comment:
I guess we need to remember to also do this with the ubuntu RPi images once we have them

Reply:
Good point. I tried a regex here but somehow I could not get it to work :sob: